bot/code_review_bot/tasks/clang_tidy.py

```diff
@@ -184,8 +184,7 @@ def display_name(self):
         return "clang-tidy"
 
     def build_help_message(self, files):
```

Comment:
I wonder if we are using the files parameter in other `build_help_message` functions. If not, we could remove it everywhere.

Reply:
We are using it for `infer` and `clang-format`.